src/Sarif.Multitool.FunctionalTests/TestData/EndLineMustNotBeLessThanStartLine/EndLineEqualsStartLine_Expected.sarif

```diff
@@ -1,46 +1,43 @@
 {
-  "$schema": "http://json.schemastore.org/sarif-1.0.0",
```

Comment:
All the _Expected files were regenerated by Update-Expected.ps1. The console output from that tool shows me that the results are correct (I also spot-checked the resulting files).